src/__tests__/screens/WordDescriptionScreen.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { shallow, } from 'enzyme'
+import { WordDescriptionScreen, } from '../../screens/WordDescriptionScreen/WordDescriptionScreen'
+
+describe('check WordDescriptionScreen', () => {
+  const changeScreen = jest.fn()
+  const deleteWord = jest.fn()
+  const props = {
+    componentId: 1233,
+    changeScreen,
+    deleteWord,
+    wordData: {
+      id: 1,
+      word: 'home',
+      transcription: 'hoʊm',
+      translation: ['дом', 'жилище'],
+      url: 'http://lokhousing.com/wp-content/uploads/2019/03/Home-1-.jpeg',
+      examples: ['Daddy went home to sleep'],
+      tagName: 'myTagList',
+    },
+  }
+
+  let wrapper
+  let instance
+  beforeEach(() => {
+    wrapper = shallow(<WordDescriptionScreen {...props} />)
+    instance = wrapper.instance()
+  })
+
+  afterEach(() => {
+    changeScreen.mockClear()
+  })
+
+  test('should render examples', () => {
+    const spy = jest.spyOn(instance, 'renderExamples')
+    instance.forceUpdate()
+
+    expect(spy).toHaveBeenCalledWith(props.wordData.examples)
+  })
+
+  test('should show loading indicator before an image will not load', () => {
```

Comment:
`should display ActivityIndicator while the image is loading` or
`should display an ActivityIndicator while loading image` ?